code/mob/living/critter/small_animal.dm

```diff
@@ -212,10 +219,48 @@ ABSTRACT_TYPE(/mob/living/critter/small_animal)
 			return
 		. = ..()
 
+	critter_attack(var/mob/target)
+		src.visible_message("<span class='combat'><B>[src]</B> bites [target]!</span>", "<span class='combat'>You bite [target]!</span>")
+		playsound(src.loc, 'sound/weapons/handcuffs.ogg', 50, 1, -1)
+		random_brute_damage(target, rand(src.attack_damage, src.attack_damage+2))
+
+	can_critter_eat()
+		src.active_hand = 2 // mouth hand
+		src.set_a_intent(INTENT_HELP)
+		return can_act(src,TRUE)
+
 /mob/living/critter/small_animal/mouse/weak
 	health_brute = 2
 	health_burn = 2
 
+/mob/living/critter/small_animal/mouse/mad
+	ai_type = /datum/aiHolder/mouse_mad
+	var/obj/machinery/wraith/rat_den/linked_den = null	//for mice spawned by plaguerat dens
```

Comment:
probably better to keep this on the `ratden` subclass. No point having normal angry mice keeping an extra empty var.